docs/content/manual/harvester-rancher/66-deploy-harvester-csi-driver-to-k3s-cluster.md

```diff
@@ -2,52 +2,68 @@
-  ![image](https://user-images.githubusercontent.com/29251855/158551034-090cee3d-9bd6-425c-84d8-16f6a66a5c64.png)
-  ![image](https://user-images.githubusercontent.com/29251855/158933473-a0172a62-5f7c-4e68-860c-c9cb38275791.png)
-  ![image](https://user-images.githubusercontent.com/29251855/158933756-d4198111-ae05-4a7d-8ea8-d21e3f3d2b87.png)
 
+1. Use the new [generate_cloud_provider_config.sh](https://github.com/harvester/harvester-csi-driver/pull/24/files) to generate `cloud-config` content for csi-driver
+    ```
+    ./generate_cloud_provider_config.sh <serviceaccount name> <namespace>
+    
+    e.g ./generate_cloud_provider_config.sh k3s default
+    
+    ```
+1. ssh to the k3s guest cluster vm
+1. Create the config-files directory
+    ```
+    mkdir -p /var/lib/rancher/rke2/etc/config-files
+    ```
 
+1. Add the cloud-provider-config file from the cloud-config part of output generated from script 
+    ```
+    vim /var/lib/rancher/rke2/etc/config-files/cloud-provider-config
+    ```
+
+1. Access k3s guest cluster on Rancher 
+1. Switch to `All Namespace`
+1. Open Apps -> Charts
+1. Install Harvester CSI Driver from the Rancher marketplace. You do not need to change the cloud-config path. 
+    ![image](https://github.com/harvester/harvester/assets/29251855/188af4e3-9735-4c59-868d-2173c1ea3971)
 
 
 ## Expected Results
-1. Can deploy K3s cluster on harvester with kubernetes version `v1.23.4+k3s1`
-  ![image](https://user-images.githubusercontent.com/29251855/158935474-9d6f1c37-ea59-485d-83f5-6f1b19ebfa98.png)
-
-1. Can correctly install harvester csi driver on K3s cluster
-  ![image](https://user-images.githubusercontent.com/29251855/158933473-a0172a62-5f7c-4e68-860c-c9cb38275791.png)
-  ![image](https://user-images.githubusercontent.com/29251855/158933756-d4198111-ae05-4a7d-8ea8-d21e3f3d2b87.png)
-
-1. Can deploy nginx service with new PVC created
-  ![image](https://user-images.githubusercontent.com/29251855/158934469-6b050d39-a45c-493d-9e29-89e16c1cf23d.png)
-  ![image](https://user-images.githubusercontent.com/29251855/158934499-6c7d9525-f43a-4426-8786-1e4aec099964.png)
-  ![image](https://user-images.githubusercontent.com/29251855/158934531-9c42b704-67cc-4c91-b24e-fa29f7183f05.png)
-1. Can allocate size in nginx container 
-  ![image](https://user-images.githubusercontent.com/29251855/158934986-c08ddccc-3b33-4508-9861-6d10c5ded3c5.png)
-1. Can resize and delete volume with harvester storage class
-  ![image](https://user-images.githubusercontent.com/29251855/158935263-a8b6fa9e-1f4b-43ae-a687-6df3e34986a1.png)
-  ![image](https://user-images.githubusercontent.com/29251855/158935288-67409de3-d5d5-4c9d-af04-421f4153658b.png)
+1. Check can successfully install csi-driver on `Installed Apps`
+  ![image](https://github.com/harvester/harvester/assets/29251855/4493d3d1-230f-4651-a9ce-cc25eae8459e)
+
+
+1. Check `Harvester` set to the `default` storage class
```

Comment:
Can you please add step to check if PVC can be created using `Harvester` storage class?

Reply:
Add test steps and expected result to rancher integration test plan 66 to check we can create new PVC using the `Harvester` storage class. 

Thanks for the review and check.